.env.example

```
@@ -1,27 +1,17 @@
-HOST_PORT=8000
-# Number of uvicorn worker processes
 NUM_OF_UVICORN_WORKERS=4
-
-# --- Ollama concurrency settings ---
-# Maximum simultaneous API requests
-CONCURRENT_REQUEST_LIMIT=2
-# Maximum parallel processes inside Ollama
-MAX_PARALLEL_PROCESSES=2
-# Maximum number of models loaded in VRAM
-OLLAMA_MAX_LOADED_MODELS=1
+HOST_PORT=8000
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The HOST_PORT key should go before the NUM_OF_UVICORN_WORKERS key

(UnorderedKey)